### DISCUSSION TITLE
Need to know where does Flowwise flow detail and data uploaded to flows are stored ?

### DISCUSSION THREAD

**Discussion by rajabhupati** [FlowiseAI/Flowise#2044] [Q&A]
By default , we see that uploaded file is stored in Database in base64 format. How does it work when we change it to blob storage when the flow is deleted /updated?

Where is the flows are stored  ?

**Comment by HenryHengZJ** [comment]
we're working on a PR - https://github.com/FlowiseAI/Flowise/pull/1976 that allow user to store into a path instead of base64 string, deleting flow will also delete every uploaded files associated with the flow
